src/sensor/adxl345b/include/eai/sensor/Adxl345bTypedefs.h

```diff
@@ -130,6 +129,22 @@ typedef struct adxl345bSensorConfiguration {
     adxl345bI2cAddress_t i2c_slave_address;
 } adxl345bSensorConfiguration_t;
 
+enum {
+    ADXL345B_SIZE_OF_RAWDATA = 6,
+};
+
+typedef union adxl345bRawData {
+    struct {
+        uint8_t xZero;
+        uint8_t xOne;
+        uint8_t yZero;
+        uint8_t yOne;
+        uint8_t zZero;
+        uint8_t zOne;
+    } DataRegisters;
+    uint8_t data[ADXL345B_SIZE_OF_RAWDATA];
+} adxl345bRawData_t;
```

Comment:
not a big deal, but I would not recommend answering the sizeof question with an enum. There is the sizeof operator for this. Of course that requires the struct that you're using to not be anonymous.